### PR TITLE
Updated travis validate translations

### PR DESCRIPTION
**Pull request by Ayub-Khan**
Updated travis to use the i18n-tools validate
to validate translations in pullrequests.
LEARNER-2556

### CODE REVIEW

**Comment by amangano-edx** [comment]
Please validate that this works by creating a test branch, committing a bad translation to it, and confirming that the build fails.

**Comment by amangano-edx** [comment]
Also, please confirm that the correct version of edx-i18n-tools is installed (>= 0.4.0) in the travis environment.

**Comment by Ayub-Khan** [comment]
I have updated to install latest version of edx_i18n_tools and also exisitng translations are failing which shows its working fine. This is blocked by https://github.com/edx/edx-ora2/pull/1091 which will fix those translations.

**Comment by Ayub-Khan** [comment]
Some other translation issues were fixed in https://github.com/edx/edx-ora2/pull/1096

File: requirements/base.txt [hunk 1]
```diff
@@ -1,4 +1,5 @@
 # edX Internal Requirements
+edx-i18n-tools
```
Comment (amangano-edx) [inline requirements/base.txt]: You probably want to ensure that the version used is >= 0.4.0, which introduced logic to exit from `validate` with a non-zero code when problems are found.

Comment (Ayub-Khan) [reply requirements/base.txt]: if we dnt specify version it automatically installs latest.

Comment (amangano-edx) [reply requirements/base.txt]: Does it do that even when it has a copy of an older version already?

File: Makefile [hunk 2]
```diff
@@ -60,7 +64,7 @@ push_translations:
 	tx push -s
 
 # extract, compile, and check if translation files are up-to-date
-validate_translations: extract_translations compile_translations generate_dummy_translations detect_changed_source_translations
+check_translations_up_to_date: extract_translations compile_translations generate_dummy_translations detect_changed_source_translations test_translations
```
Comment (amangano-edx) [inline Makefile]: Should `test_translations` still be in here?

Comment (Ayub-Khan) [reply Makefile]: no. sorry my bad.

File: requirements/base.txt (hunk 1 above)
Comment (Ayub-Khan) [reply requirements/base.txt]: @amangano-edx Im not sure how travis handles that. But before that it was using 0.3.9 now its using 0.4.3 which can be seen here https://travis-ci.org/edx/edx-ora2/jobs/362037159

**Pull request merged**